src/ConnectedMode/Binding/Suggestion/SuggestSharedBindingGoldBar.cs

```diff
@@ -53,6 +53,10 @@ public SuggestSharedBindingGoldBar(INotificationService notificationService,
             this.doNotShowAgainNotificationAction = doNotShowAgainNotificationAction;
             this.solutionInfoProvider = solutionInfoProvider;
             this.browserService = browserService;
+            
+            // Prevent INotificationService from handling the on solution change events
+            // as this is exceptionally overriden by the SharedBindingSuggestionService
+            this.notificationService.CloseOnSolutionClose = false;
```

Comment:
Not a fan of this approach of setting the value directly on the service instead of the notification